src/World.h

```diff
@@ -464,6 +464,10 @@ class cWorld :
 	Returns the UniqueID of the spawned experience orb, or cEntity::INVALID_ID on failure. */
 	virtual UInt32 SpawnExperienceOrb(double a_X, double a_Y, double a_Z, int a_Reward) override;
 
+	/** Spawns an experience orb at the given location with the given reward, by default split into multiple experience orbs if possible,
+	Returns spawned experience orbs UniqueID */
+	virtual std::vector<UInt32> SpawnSplitExperienceOrbs(double a_X, double a_Y, double a_Z, int a_Reward, bool a_Split = true);
```

Comment:
Remove the `bool`, this function should always split because it's part of the name.